services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -11,15 +11,20 @@ import {
   ProfilePage,
   ReportPageWrapper,
   ReviewSubmitPage,
+  ExportedReportPage,
 } from "components";
 // utils
 import { ScrollToTopComponent, useStore } from "utils";
 // types
 import { ReportRoute, ReportType } from "types";
 
 export const AppRoutes = () => {
-  const { userIsAdmin } = useStore().user ?? {};
+  const { userIsAdmin, userReports } = useStore().user ?? {};
```

Comment:
I don't understand the purpose of `userReports` 🤔

Reply:
Pretty sure we don't need `userReports` because all users have access to both reports in MFP (as opposed to MCR where in the past we've had only some users have access to certain reports).